Dockerfile

```diff
@@ -22,15 +26,20 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o /tmp/rustup-ini
     chmod +x /tmp/rustup-init && \
     /tmp/rustup-init -y --no-modify-path \
         --default-toolchain nightly-2022-01-13 \
-        -t thumbv7em-none-eabihf && \
+        -t thumbv7em-none-eabihf \
+	-c llvm-tools-preview \
```

Comment:
If `-c llvm-tools-prevew` is the option for `/tmp/rustup-init` then could be aligned with others?

Reply:
Fixed.

Reply:
Not fixed, this is not in the same column as `-t ...` above.